ScrollableTabBar.js

```diff
@@ -59,6 +59,9 @@ const ScrollableTabBar = React.createClass({
   },
 
   updateView(offset) {
+    if (offset.value === undefined) {
```

Comment:
This one is better
```js
if (!offset.value) {
```

Reply:
Applied in c595108c71bf864e2cfa047affe3883d74fa49d9